app/src/main/res/values/strings.xml

```diff
@@ -166,4 +166,13 @@
     <string name="re_authenticate_fail">"Failed to re-authenticate!"</string>
     <string name="update_username_success">"Successfully updated username</string>
     <string name="update_username_fail">Failed to update username</string>
+    <string name="add_history">Add to history</string>
+    <!-- Strings used for line chart    -->
+    <string name="results">Results</string>
+    <string name="weight_unit">kg</string>
+    <string name="body_fat_percentage_unit">%</string>
```

Comment:
If you use a symbol that is going to be the same for all languages you can do the following to declare it only once in the default `strings.xml`:
```suggestion
    <string name="body_fat_percentage_unit" translatable="false">%</string>
```